.goreleaser.yml

```diff
@@ -15,6 +15,29 @@ archives:
       - completions/*
       - manpages/*
 
+aurs:
+  - homepage:  https://www.updatecli.io
+    description: Updatecli is a declarative dependency management command line tool.
```

Comment:
`A declarative dependency management command line tool.` is sufficient :smile:

Reply:
Thanks for the feedback

Reply:
```suggestion
    description: A declarative dependency management tool.
```